LibertyDSNP/example-client
# Feature/setup server 178145869

Purpose
---------------
As a developer, I want to have a backend server for storing content and batch files for data generated from the example client.
We just want to get it setup and running in the background whenever the example client is started as well.


Solution
---------------
Restructured example client file structure so that we now have client and example server directories. Got an express server setup in the server directory. It is modelled after Carbonfive Express Spraygun and Project Liberty exp-server


Change summary
---------------
* Separated Client and Server into to folders named accordingly.
* Added Express Server 
* Fixed Github actions to test both
* Added app.ts and server.ts
* Included simple "Hello World" endpoint at root


Steps to Verify
----------------
1. Setup example client
2. Change to server folder
3. Run `npm run watch`
4. Server should startup without issue
5. Navigating to endpoint localhost:3001 should display Hello World
